runtime/moonbeam/src/lib.rs

```diff
@@ -975,6 +975,9 @@ impl Contains<RuntimeCall> for NormalFilter {
 				pallet_assets::Call::approve_transfer { .. } => true,
 				pallet_assets::Call::transfer_approved { .. } => true,
 				pallet_assets::Call::cancel_approval { .. } => true,
+				pallet_assets::Call::destroy_accounts { .. } => true,
```

Comment:
Not very familiar with this part but reading Substrate doc:

> `destroy_accounts` should only be called after `start_destroy` has been called, and the asset is in a `Destroying` state

Same applies to `destroy_approvals` and `finish_destroy`. Meaning that we either filter out all `destroy*` calls, or we also allow `start_destroy`. cc @girazoki

Reply:
[`start_destroy`](https://paritytech.github.io/substrate/master/pallet_assets/pallet/struct.Pallet.html#method.start_destroy) is a special case that we use so `pallet-asset-manager` can handle the lifecycle of an asset, since it allows the use of the `ForceOrigin` which is the asset manager. Once invoked the asset's status is changed to `Destroying` and it can never become `Live` again. All other calls do not use `ForceOrigin` and are callable by any `Signed` origin and hence are treated differently. This was discussed with @girazoki.